### ISSUE TITLE
'getProfile' Deprecated?

### ISSUE THREAD

**Issue by jtsom** [auth0/lock#725]
Otherwise, create a new issue with the following information:
-------------------------------------------------------------

- [x] Code snippet or sample project that reproduces the bug
```
this.lock.on('authenticated', (authResult) => {
      localStorage.setItem('id_token', authResult.idToken);
      this.lock.getProfile(authResult.idToken, (error, profile) => {
        console.log(profile);
        this.userProfile = profile;
        localStorage.setItem('profile', JSON.stringify(profile));

        // redirect if there is a saved url
        let redirectUrl: string = localStorage.getItem('redirect_url');
        if (redirectUrl !== undefined && redirectUrl) {
          this.router.navigate([redirectUrl]);
          localStorage.removeItem('redirect_url');
        }
      });
    });
```

- [ ] Lock version
current - `"auth0-lock": "^10.6.1"`
- [ ] Browser & OS

Logging in with Auth0 Lock and when the authenticated callback fires, the call to `lock.getProfile` generates a deprecation warning:

`"DEPRECATION NOTICE: This method will be soon deprecated, use `getUserInfo` instead."`

Apparently, the `getProfile` routine on the Lock object, calls down to the Auth0 `getProfile` routine, which was deprecated.

**Comment by hzalaz** [comment]
It's not deprecated itself but a method of auth0.js it relies on. We'll see if we remove the console.warn or provide an alternative

**Comment by jtsom** [comment]
Well, I'd say, the Lock .getProfile routine should call down to the Auth0 .getUserInfo routine, instead of the Auth0 deprecated .getProfile routine.

**Comment by hzalaz** [comment]
No because that will be a breaking change since userInfo requires the `access_token`

**Comment by dimitrovs** [comment]
The issue is closed but I still see the warning with auth0-7.4 and lock 10.6 . Was it fixed in a newer version of one of these two libraries?
